vpc/main.tf

```diff
@@ -103,6 +103,17 @@ resource "aws_nat_gateway" "public" {
   )
 }
 
+resource "aws_ssm_parameter" "nat_gateway_eip" {
+  # checkov:skip=CKV_AWS_337:Ensure SSM parameters are using KMS CMK. Related ticket: https://uktrade.atlassian.net/browse/DBTP-946
+  # checkov:skip=CKV2_AWS_34:AWS SSM Parameter should be Encrypted. Related ticket: https://uktrade.atlassian.net/browse/DBTP-946
+  for_each = toset(var.arg_config.nat_gateways)
+  name     = "/${var.arg_name}/nat-eip-${each.key}/ADDITIONAL_IP_LIST"
```

Comment:
@ejayesh in the ticket (https://uktrade.atlassian.net/jira/software/c/projects/DBTP/boards/458?quickFilter=882&selectedIssue=DBTP-946) it says to call it /ADDITIONAL_IP_LIST, but also in the ticket it says that we are not to add the parameter to a list.
Any clarification around this please?